modules/motions/ui/MotionFormStartNew/Parts/StartSDVTNodeOperatorsAdd.tsx

```diff
@@ -33,34 +38,26 @@ type NodeOperator = {
   managerAddress: string
 }
 
-export const ALLOWED_SDVT_NODE_OPERATORS_ADD_MAP = {
-  [MotionTypeForms.SDVTNodeOperatorsAdd]: {
-    evmContract: ContractSDVTNodeOperatorsAdd,
-    motionType: MotionTypeForms.SDVTNodeOperatorsAdd,
-  },
-}
-
 // DONE: The current number of node operators in the registry MUST be equal to the _nodeOperatorsCount
 // DONE: (exec also) The total number of node operators in the registry, after adding the new ones, MUST NOT exceed nodeOperatorsRegistry.MAX_NODE_OPERATORS_COUNT()
 // DONE: Manager addresses MUST NOT have duplicates
-// TODO: Manager addresses MUST NOT be used as managers for previously added node operators
-// TODO: Reward addresses of newly added node operators MUST NOT contain the address of the stETH token
+// DONE: Manager addresses MUST NOT be used as managers for previously added node operators
+// DONE: Reward addresses of newly added node operators MUST NOT contain the address of the stETH token
 // DONE: Reward addresses of newly added node operators MUST NOT contain zero addresses
 // DONE: The names of newly added node operators MUST NOT be an empty string
 // DONE: The name lengths of each newly added node operator MUST NOT exceed the nodeOperatorsRegistry.MAX_NODE_OPERATOR_NAME_LENGTH()
 export const formParts = ({
   registryType,
 }: {
-  registryType: keyof typeof ALLOWED_SDVT_NODE_OPERATORS_ADD_MAP
+  registryType: typeof MotionTypeForms.SDVTNodeOperatorsAdd
```

Comment:
I think we can get rid of `registryType` and just use constant `motionType.SDVTNodeOperatorsAdd` along the way since this form isn't a generic one.